modules/ROOT/pages/salesforce/salesforce-connector-96.adoc

```diff
@@ -0,0 +1,1211 @@
+. Click the Job ID to view the job detail page.
+
+The job detail page includes a related list of all the batches for the job. The related list provides View Request and View Response links for each batch. If the batch is a CSV file, the links return the request or response in CSV format. If the batch is an XML file, the links return the request or response in XML format. These links are available for batches created in Salesforce API version 19.0 and later.
+
+== Usage Notes
+
+=== Fields To Null
+
+* The configurations have a checkbox called Can Clear Fields by Updating Field Value to Null. If checked, all the fields in a request that have a Null value are added to the `fieldsToNull` field and sent to Salesforce.
+
+* You can decide which fields to set to Null without being forced to use the `fieldsToNull` field.
+
+=== Upsert
+
+* Unless you configure the External ID Field Name for the sObject to which you are trying to Upsert, every use of the upsert will fail.
+* The Upsert operation does not work with the sObject `priceBookentry2`.
+* While you can't change the contentType for bulk upsert, you can use Create Job
+to set the contentType to either
+CSV or zipped CSV (if you're hitting the character limit). Follow up with Create Batch.
+
+=== Query
+
+* Even though you can see the fields of an sObject and their corresponding types via DataSense, the Query operation returns _all_ fields as `String`.
+
+* If you want to use the actual type of the field, you must convert that field to the desired type using a Transform (or Transform Message) component.
+
+* Although `CreatedDate` field appears as `dateTime`, the query returns a String representing the date.
+
+* To actually use the field as a dateTime, you can configure it using Transform Message, like in the example.
+
+* To store Date and DateTime fields, you can use DataWeave expressions to create Date and Calendar Java objects.
+
+=== Insert values into a Salesforce Drop-down
+
+* Be aware that inserting dependent values into an existing drop-down list field in Salesforce does not always work. Test to confirm functionality.
+
+=== Evaluate Values in a Salesforce Drop-down
+
+* If you are evaluating against a value in an existing drop-down list field in Salesforce, be sure to use the _exact value_ in the dropdown. For example, if you use the value "US" to evaluate against the contents of a drop-down list that contains the value "USA", the evaluation works, but you end up with two values in the dropdown: one for US and one for USA.
+
+=== Currency
+
+* Currency values cannot exceed 18 characters in length.
+* When working with multiple currencies, be aware of which currency your sObject uses so that you avoid inaccurate entries. The default currency matches the location at the organization level.
+
+=== Limits on API Calls
+
+* You need to know the rate limiting policy that applies to your account so that you do not exceed the number of allotted API calls per day.
```

Comment:
Salesforce has both concurrent and org-wide per 24-hour. https://developer.salesforce.com/docs/atlas.en-us.salesforce_app_limits_cheatsheet.meta/salesforce_app_limits_cheatsheet/salesforce_app_limits_platform_api.htm and